src/manifolds/Stiefel.jl

```diff
@@ -90,9 +91,9 @@ function check_tangent_vector(
         kwargs...,
     )
     mpv === nothing || return mpv
-    if !isapprox(p' * X + X' * p, zeros(k, k); kwargs...)
+    if !isapprox(p' * X + conj(X' * p), zeros(k, k); kwargs...)
```

Comment:
Wouldn't this be safer, since equality checks with zeros often require much stricter tolerances (so with default tolerances these will often fail)?
```suggestion
    if !isapprox(p' * X, -conj(X' * p); kwargs...)
```

Reply:
nice, sure.